i3-wk-switch.py

```diff
@@ -1,6 +1,6 @@
-#!/usr/bin/python
+#!/usr/bin/env python
 
-"""Emulates xmonad's workspace switching behavior in i3"""
+"""Emulates xmonad's workspace switching behavior in i3 and sway"""
 
 # pylint: disable=no-member,invalid-name
```

Comment:
Add `too-many-function-args` to suppress warning

Reply:
I don't get a warning?